### PR TITLE
virttest/utils_libguestfs.py

### PR DESCRIPTION
**Pull request by zcyes**
add file_attr_ops

### CODE REVIEW

**Comment by zcyes** [comment]
add file "guestfish_fs_attr_ops.py/cfg" for Libguestfs fis_attr_ops test, and the commands are: (set_e2uuid, get_e2uuid, vfs_uuid, findfs_uuid) (set_uuid) (set_e2label, get_e2label, vfs_label, findfs_label) (set_label) (set_e2attrs, get_e2attrs) (set_e2generation, get_e2generation) (statvfs) (tune2fs_l) (tune2fs) (vfs_type, fsck) (mkfs/mkfs_opts) (blkid) (e2fsck) (filesystem_available) (list_filesystems) (mkfifo) (sync) (mklost_and_found) (mknod) (mknod_b) (mknod_c) (ntfsresize_opts)

**Comment by zcyes** [comment]
1. add test (resize2fs, resize2fs_M, resize2fs_size)
2. support different filesystems

File: libguestfs/tests/guestfish_fs_attr_ops.py [hunk 1]
```diff
@@ -0,0 +1,1123 @@
...
+    gf.run()
+
+    mount_point = params.get("mount_point")
+    gf.mount(mount_point, '/')
+
+    nodename = '/testnode'
+    gf.rm_rf(nodename)
+    gf.umask('022')
+
+    # mknod_b
```
Comment (shi2wei3) [inline libguestfs/tests/guestfish_fs_attr_ops.py]: typo here, and major number 8 is not a char device

File: libguestfs/tests/guestfish_fs_attr_ops.py [hunk 2]
```diff
@@ -0,0 +1,1123 @@
...
+    mode = "0060777"
+    major = "8"
+    minor = "1"
+    permission = "brwxr-xr-x"
```
Comment (shi2wei3) [inline libguestfs/tests/guestfish_fs_attr_ops.py]: Please check the src, look like this API doesn't have constraints to check whether your assigned a major number belongs to block device.

**Comment by shi2wei3** [comment]
Test pass for the test cases in this PR, code looks good to me now.

**Pull request merged**